src/components/paragraphs/sum-pill-banner/sum-pill-banner-paragraph.tsx

```diff
@@ -20,22 +20,12 @@ const SumPillBannerParagraph = ({paragraph, ...props}: Props) => {
   return (
     <Element
       {...props}
-      className={twMerge("relative left-1/2 w-screen -translate-x-1/2 py-60", props.className)}
+      className={twMerge("relative left-1/2 w-screen -translate-x-1/2", props.className)}
       aria-labelledby={paragraph.sumPillBannerHeadline ? paragraph.id : undefined}
     >
       {paragraph.sumPillBannerBkgd && (
-        <div
-          className={twMerge(
-            "absolute left-0 top-0 -z-10 h-full w-full bg-poppy-light",
-            clsx({
-              "bg-olive": behaviors.sum_pill_banner_behaviors?.sum_pill_banner_overlay_bkg === "olive",
-              "bg-spirited text-black-true": behaviors.sum_pill_banner_behaviors?.sum_pill_banner_overlay_bkg === "spirited",
-              "bg-white": behaviors.sum_pill_banner_behaviors?.sum_pill_banner_overlay_bkg === "white",
-              "bg-spirited-dark": behaviors.sum_pill_banner_behaviors?.sum_pill_banner_overlay_bkg === "spirited_dark",
-            })
-          )}
-        >
-          <div className="relative top-1/2 h-1/2 w-full">
+        <div className="absolute left-0 top-0 -z-10 h-full w-full">
+          <div className="relative h-full w-full">
```

Comment:
Where'd all the color options go?

Reply:
Background banner colors are still here on [L42-52](https://github.com/SU-SWS/summer-nextjs/pull/22/files/f82716c99855fd3927aa0454ac0b992d8a2a9085#diff-202ee33c3e67e922391e251c236c22cb14f442bfa4290735a9279e1cdb041effR42-R52) 😃

I adjusted things a bit so that the colors are part of the banner content instead of the image. Was there a reason why we needed the colors options to be defined in two different areas? Was it to account for when the content author did not select an image?

Reply:
Oh ok, I don't exactly remember why i put it in both places, but you could be correct. But without an image looks ok with this PR
![image](https://github.com/SU-SWS/summer-nextjs/assets/7185045/bb191691-0bd9-4a17-8e47-890a130b6556)